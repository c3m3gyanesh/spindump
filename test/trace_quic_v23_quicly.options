--format json --report-notes --report-spins
