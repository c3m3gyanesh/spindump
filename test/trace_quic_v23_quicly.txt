A test between Picoquic client and the Fastly "Quicly" implementation. There were two tests (plain QUIC and HTTP23), but the server was not responding on the latter. The commands used were:

    ./picoquic/picoquicdemo -l ./test.log -L -a "hq-23" -v ff000017 quic.examp1e.net 4433 /5000
    ./picoquic/picoquicdemo -l ./test.log -L -a "h3-23" -v ff000017 h2o.examp1e.net 8443 /5000
