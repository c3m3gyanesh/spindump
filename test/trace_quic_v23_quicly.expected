[
{ "Event": "new", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "087bed967a6ea318-c454e592f7cc1682 (51120:4433)", "Ts": 1571181049671445, "State": "Starting", "Notes": "No rsp,v23,no spin", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "087bed967a6ea318-0870152eb7589b39 (51120:4433)", "Ts": 1571181049805547, "State": "Up", "Notes": "No rsp,v23,no spin", "Right_rtt": 134102, "Packets1": 1, "Packets2": 0, "Bytes1": 1280, "Bytes2": 0, "Bandwidth1": 1280, "Bandwidth2": 0 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "7bed967a6ea318cc-0870152eb7589b39 (51120:4433)", "Ts": 1571181049807215, "State": "Up", "Notes": "V23,no spin", "Value": 1, "Who": "initiator", "Packets1": 3, "Packets2": 3, "Bytes1": 1581, "Bytes2": 3530, "Bandwidth1": 1581, "Bandwidth2": 3530 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "7bed967a6ea318cc-0870152eb7589b39 (51120:4433)", "Ts": 1571181049807216, "State": "Up", "Notes": "V23,no spin", "Value": 1, "Who": "initiator", "Packets1": 4, "Packets2": 3, "Bytes1": 1664, "Bytes2": 3530, "Bandwidth1": 1664, "Bandwidth2": 3530 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "7bed967a6ea318cc-0870152eb7589b39 (51120:4433)", "Ts": 1571181049937429, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 5, "Packets2": 4, "Bytes1": 3132, "Bytes2": 3606, "Bandwidth1": 3132, "Bandwidth2": 3606 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "7bed967a6ea318cc-0870152eb7589b39 (51120:4433)", "Ts": 1571181049937550, "State": "Up", "Notes": "V23,no spin", "Value": 1, "Who": "initiator", "Packets1": 5, "Packets2": 5, "Bytes1": 3132, "Bytes2": 4914, "Bandwidth1": 3132, "Bandwidth2": 4914 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "7bed967a6ea318cc-0870152eb7589b39 (51120:4433)", "Ts": 1571181049938458, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 6, "Packets2": 5, "Bytes1": 3215, "Bytes2": 4914, "Bandwidth1": 3215, "Bandwidth2": 4914 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "7bed967a6ea318cc-0870152eb7589b39 (51120:4433)", "Ts": 1571181049939543, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 6, "Packets2": 6, "Bytes1": 3215, "Bytes2": 6222, "Bandwidth1": 3215, "Bandwidth2": 6222 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "7bed967a6ea318cc-0870152eb7589b39 (51120:4433)", "Ts": 1571181049939634, "State": "Up", "Notes": "V23,no spin", "Value": 1, "Who": "initiator", "Packets1": 6, "Packets2": 7, "Bytes1": 3215, "Bytes2": 7530, "Bandwidth1": 3215, "Bandwidth2": 7530 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "7bed967a6ea318cc-0870152eb7589b39 (51120:4433)", "Ts": 1571181049940892, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 7, "Packets2": 7, "Bytes1": 3298, "Bytes2": 7530, "Bandwidth1": 3298, "Bandwidth2": 7530 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "7bed967a6ea318cc-0870152eb7589b39 (51120:4433)", "Ts": 1571181049941139, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 7, "Packets2": 8, "Bytes1": 3298, "Bytes2": 8838, "Bandwidth1": 3298, "Bandwidth2": 8838 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "7bed967a6ea318cc-0870152eb7589b39 (51120:4433)", "Ts": 1571181049941362, "State": "Up", "Notes": "V23,no spin", "Value": 1, "Who": "initiator", "Packets1": 7, "Packets2": 9, "Bytes1": 3298, "Bytes2": 8905, "Bandwidth1": 3298, "Bandwidth2": 8905 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "7bed967a6ea318cc-0870152eb7589b39 (51120:4433)", "Ts": 1571181049967200, "State": "Up", "Notes": "V23,no spin", "Value": 0, "Who": "responder", "Packets1": 8, "Packets2": 9, "Bytes1": 3365, "Bytes2": 8905, "Bandwidth1": 3365, "Bandwidth2": 8905 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "7bed967a6ea318cc-0870152eb7589b39 (51120:4433)", "Ts": 1571181049967422, "State": "Up", "Notes": "V23,no spin", "Value": 1, "Who": "initiator", "Packets1": 8, "Packets2": 10, "Bytes1": 3365, "Bytes2": 8965, "Bandwidth1": 3365, "Bandwidth2": 8965 },
{ "Event": "new", "Type": "QUIC", "Addrs": ["172.16.114.251","133.242.206.244"], "Session": "0839f070e4d69a74-7bfcc06db50bc337 (53457:8443)", "Ts": 1571181083978213, "State": "Starting", "Notes": "No rsp,v23,no spin", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 }
]
